app/javascript/src/utils/index.js

```diff
@@ -1,3 +1,18 @@
 import * as R from "ramda";
 
+const camelize = value =>
+  value
+    .toLowerCase()
+    .split(" ")
+    .reduce(
+      (word, character) =>
+        word + (character.charAt(0).toUpperCase() + character.slice(1))
+    );
```

Comment:
So what is the output if I give the value as `Ux-Designer` or some other string with non-alphanumeric chars? My expectation is the camelized string will be `uxDesigner`.

Reply:
If the value given will be "Ux-Designer", the output will be "ux-designer". Yes, it's not correct. Thank you for pointing that mistake. 
It can be  corrected by using RegExp. 
I will make the changes in the pattern of the split method to use regular expression.

So the new code will look like :-

const camelize = value =>
  value
    .toLowerCase()
    .split(/[-_,./:\s]/g)
    .reduce(
      (word, character) =>
        word + (character.charAt(0).toUpperCase() + character.slice(1))
    );

Reply:
@aryanandarnav13 Got it. By the way, you can syntax highlight code blocks in Github comments, since it's basically markdown. Refer: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

Reply:
Thank you for sharing this link for highlighting code blocks, I will bring this to my practice from now onwards.